src/components/CustomerContext.js

```diff
@@ -101,13 +108,37 @@ class CustomerProvider extends Component {
     }
   };
 
+  getCampaigns = async () => {
+    try {
+      this.setState({ fetchingCampaigns: true });
+      let campaigns = await fetch(
+        `${process.env.REACT_APP_API_URL}/campaigns`,
+        {
+          include: "credentials",
+        }
+      ).then((camps) => camps.json());
+
+      campaigns = campaigns.campaigns.filter(
```

Comment:
this is misleading. let's not use `let`s unless it's unavoidable.
here you unwrap the object, filter its content, and save it as `campaigns` again.

How about this?
```
const { campaigns } = await fetch(...)
const campaignsFiltered = campaigns.filter()...
```

or

```
const allCampaigns = await fetch()
const campaigns = allCampaigns.campaigns.filter()
```